### PR TITLE
Petolon Presence

### PR DESCRIPTION
**Pull request by TheGamerzs**

### CODE REVIEW

**Comment by TheGamerzs** [comment]
Im sorting the deepscan issues

**Comment by TheGamerzs** [comment]
Do Not Push

**Comment by TheGamerzs** [comment]
Ready for Push

**Comment by doomlerd** [comment]
screenshots

**Comment by TheGamerzs** [comment]
image
image
image
image

**Comment by doomlerd** [comment]
the type you're using is wrong, PresenceData is correct.

File: websites/P/Peloton/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+    "author": {
+        "name": "The Gamerzs",
+        "id": "712294190339588209"
+    },
+    "url": [
```
Comment (eggsywashere) [inline websites/P/Peloton/dist/metadata.json:6]: Single URL, please use string instead of an array.

File: websites/P/Peloton/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        "www.onepeloton.com"
+    ],
+    "description": {
+        "en": "Peloton is attractive, addictive, and seriously whips you into shape."
+    },
+    "regExp": "([a-z0-9-]+[.])*onepeloton([.][a-z]|[co.uk])",
```
Comment (Alanexei) [inline websites/P/Peloton/dist/metadata.json:12]: image
I think you may need to readjust your regex, unless this is correct behavior.

Comment (TheGamerzs) [reply websites/P/Peloton/dist/metadata.json:12]: TLD change between countries but have the same file names and file structure

Comment (TheGamerzs) [reply websites/P/Peloton/dist/metadata.json:12]: Subdomains are too many to specify

Comment (Bas950) [reply websites/P/Peloton/dist/metadata.json:15]: `([a-z0-9-]+[.])*onepeleton([.][a-z]+)+[/]`

Comment (Bas950) [reply websites/P/Peloton/dist/metadata.json:15]: image

File: websites/P/Peloton/dist/metadata.json (hunk 1 above)
Comment (TheGamerzs) [reply websites/P/Peloton/dist/metadata.json:6]: Added other urls

**Pull request merged**